system/modules/timelog/partials/templates/timelogwidget.tpl.php

```diff
@@ -82,11 +113,11 @@ function saveTimer() {
                     success: function(data) {
                         var object_data = JSON.parse(data);
                         
-                        start_time = (new Date().getTime() / 1000);
+                        start_time = object_data.start_time ? object_data.start_time : (new Date().getTime() / 1000);
```

Comment:
Hmm the Chrome console doesn't seem to care much either... maybe I'm thinking about something else, either way both are correct as if object_data.start_time doesn't exist, it will equal ```undefined```.